src/system/PredefinedProcedure.cpp

```diff
@@ -8,17 +8,36 @@
 #include <string>
 #include <utility>
 #include <vector>
+#include <iostream>
 
 using std::make_unique;
 using std::pair;
 using std::string;
 using std::vector;
+using std::cout;
```

Comment:
`cout` does not seem to be used, nor should it!

Reply:
Cruft left from debugging. Removed.